code/modules/uplink/uplink_item.dm

```diff
@@ -1051,6 +1052,49 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/organ/cyberimp/chest/reviver
 	cost = 8
 
+/datum/uplink_item/cyber_implants/esword	//This is a bad idea
+	name = "Arm Energy-Blade Projector"
+	desc = "An illegal cybernetic combat implant that allows the user to project a blade of lethal energy from their hand."
+	item = /obj/item/organ/cyberimp/arm/esword
+	cost = 12
+	player_minimum = 30
+
+/datum/uplink_item/cyber_implants/esword/traitor	//This is a worse idea
+	name = "Arm Energy-Blade Projector"
+	desc = "An illegal cybernetic combat implant that allows the user to project a blade of lethal energy from their hand."
+	item = /obj/item/organ/cyberimp/arm/esword
+	cost = 16
+	include_modes = list()
+	exclude_modes = list(/datum/game_mode/nuclear)
+	player_minimum = 30
+
+/datum/uplink_item/cyber_implants/medibeam
+	name = "Medical Beam Arm Implant"
+	desc = "An cybernetic implant that gives the user an extendable medical beamgun in their arm"
+	item = /obj/item/organ/cyberimp/arm/medibeam
+	cost = 20
+	player_minimum = 15
+
+/datum/uplink_item/cyber_implants/flash
```

Comment:
The implants NEED to use the one use variant of the autoimplanter if they are going to be in the traitor uplink. Since for 6 tc you can bypass the need for surgery for as many implants as you want.